modules/admin/proc-configuring-postgresql-with-tls-using-helm.adoc

```diff
@@ -0,0 +1,132 @@
+[id="proc-configuring-postgresql-with-tls-using-helm_{context}"]
+= Configuring an external PostgreSQL instance with TLS connection using the Helm Chart
+
+You can edit the Helm Chart default setting to configure an external PostgreSQL database server with TLS connection.  
+
+.Prerequisites
+
+* You have the following details:
+** `db-host`: Denotes your PostgreSQL instance Domain Name System (DNS) or IP address 
+** `db-port`: Denotes your PostgreSQL instance port number, usually `5432`
+** `username`: Denotes the user name to connect to your PostgreSQL instance
+** `password`: Denotes the password to connect to your PostgreSQL instance
+
+* You have the certificate files and the TLS private key. For more information, see link:{LinkAdminGuide}#postgresql-certificate-for-crunchy-database_admin-rhdh[Generating PostgreSQL certificate for the Crunchy database].
+
+.Procedure
+
+. Create a credential secret named `postgres-cred` to connect with the PostgreSQL instance:
++
+[source,yaml]
+----
+cat <<EOF | oc -n <your-namespace> create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+ name: postgres-cred
+data: <1>
+  POSTGRES_PASSWORD: <password>
+  POSTGRES_PORT: "<db-port>"
+  POSTGRES_USER: <username>
+  POSTGRES_HOST: <db-host>
+  PGSSLMODE: <ssl-mode> # for TLS connection <2>
+EOF
+----
+<1> Provide credential data to connect with your PostgreSQL instance.
+<2> Provide the value to configure a TLS connection for your PostgreSQL instance.
+
+. Create a certificate secret named `postgress-external-db-cluster-cert` with the CA certificate, TLS key, and TLS certificate files:
```

Comment:
The step of creating a certificate should precede the creation of a credential secret

This is relevant for all the pages where you create credential secret and a certificate